app/design/frontend/Magebit/learning/Magento_Catalog/templates/product/view/attributes.phtml

```diff
@@ -1,35 +1,44 @@
 <?php
 /**
- * Template for displaying selected product attributes and short description.
- * @var \Magento\Catalog\Block\Product\View\Attributes $block
- * @var \Magento\Catalog\Model\Product $product
+ * @copyright Copyright (c) 2024 Magebit, Ltd. (https://magebit.com/)
+ * @author    Magebit <info@magebit.com>
+ * @license   MIT
  */
 
-$product = $block->getProduct();
-$attributes = $product->getAttributes();
-
-/** @var \Magento\Catalog\Helper\Output $catalogHelper */
-$catalogHelper = $this->helper('Magento\Catalog\Helper\Output');
+declare(strict_types=1);
+use Magebit\Learning\Block\Product\View\Attributes;
+use Magento\Framework\Escaper;
+use Magento\Framework\Exception\LocalizedException;
 
-$displayAttributes = ['dimensions', 'color', 'material'];
+/**
+ * Template for displaying selected product attributes and short description.
+ * @var Attributes $block
+ * @var Escaper $escaper
+ */
 ?>
 
 <div class="product-details">
-    <?php foreach ($displayAttributes as $code): ?>
-        <?php if (isset($attributes[$code])): ?>
-            <?php $attribute = $attributes[$code]; ?>
+    <?php $attributes = $block->getDisplayAttributes(); ?>
```

Comment:
If you wan't to declare some unified variable, better initialize at the beginnign of the template if possible.
Something like :
```

/**
 * Template for displaying selected product attributes and short description.
 * @var Attributes $block
 * @var Escaper $escaper
 */

$attributes = $block->getDisplayAttributes();
?>

//Beginyour html template
```